std/experimental/allocator/building_blocks/allocator_list.d

```diff
@@ -661,65 +666,271 @@ version (Posix) @system unittest
     assert(b1.length == 1024 * 10);
 }
 
+/// Ditto
+shared struct SharedAllocatorList(Factory, BookkeepingAllocator = GCAllocator)
+{
+    import std.typecons : Ternary;
+    import std.traits : hasMember;
+    import core.internal.spinlock : SpinLock;
+
+private:
+    // Forward all calls to 'impl' and protect them by the lock below
+    AllocatorList!(Factory, BookkeepingAllocator) impl;
+    SpinLock lock = SpinLock(SpinLock.Contention.brief);
+
+    // Remove this when 'https://github.com/dlang/druntime/pull/2156' gets merged
```

Comment:
does this have a corresponding dmd PR? is it still open?

Reply:
I haven’t been able to find any successor to that druntime PR.

Reply:
Should we rather rephrase that note or simply remove it?

Reply:
Theoretically, there’s https://github.com/dlang/dmd/issues/17128.

Reply:
So, I’ve replaced this comment in 45a094f57629f90e9f8bca82fc55b5ee4a6811aa.